reggie/configs/configs.py

```diff
@@ -21,9 +21,10 @@ def __init__(self, state=None, file_name=None):
         self.data = self.load_data(config_file, self.infer_locale_file(
             config_file))
 
-        self.primary_locale_column = self.data[PRIMARY_LOCALE_ALIAS]
-        self.primary_locale_type = self.data.get(PRIMARY_LOCALE_TYPE, "county")
-        self.primary_locale_names = self.data[PRIMARY_LOCALE_NAMES]
+        if self.data[PRIMARY_LOCALE_NAMES] is not None:
+            self.primary_locale_column = self.data[PRIMARY_LOCALE_ALIAS]
```

Comment:
if you switch your county identfiers to primary locale identifers, you can leave these 2 lines out of the conditional then